model/src/payout_curve.rs

```diff
@@ -158,13 +182,21 @@ struct PayoutParameter {
 }
 
 impl PayoutParameter {
-    fn into_payouts(self) -> Result<Vec<Payout>> {
+    fn into_payouts_long_taker(self) -> Result<Vec<Payout>> {
         generate_payouts(
             self.left_bound..=self.right_bound,
             bitcoin::Amount::from_sat(self.short_amount),
             bitcoin::Amount::from_sat(self.long_amount),
         )
     }
+
+    fn into_payouts_short_taker(self) -> Result<Vec<Payout>> {
```

Comment:
Why is this necessary? If the caller know the role of the position, wouldn't that be good enough?

Reply:
It is not necessary but convenient. Before we had an `into_payouts` function which knew that the taker was always long. 

I can remove both functions and inline this logic.

Reply:
Shouldn't we change `maia` to care about `long` and `short` instead of `maker` and `taker`?

I am happy to merge this for now but we should fix this in maia I believe.

Reply:
I created a ticket for it: https://github.com/comit-network/maia/issues/16

Reply:
> Shouldn't we change `maia` to care about `long` and `short` instead of `maker` and `taker`?

Ultimately yes.